.github/workflows/regression-report.yml

```diff
@@ -0,0 +1,251 @@
+#####################################################################################
+# GitHub Action to generate Checkstyle report.
+#
+# Workflow starts when:
+# 1) PR comment - created
+#
+# Requirements:
+# 1) secrets.AWS_ACCESS_KEY_ID - access key for AWS S3 service user
+# 2) secrets.AWS_SECRET_ACCESS_KEY - security access key for AWS S3 service user
+#
+# If you need to change bucket name or region, change AWS_REGION and AWS_BUCKET_NAME variables.
+# For another bucket, you will need to change the secrets.
+#####################################################################################
+name: regression-report
+env:
+  AWS_REGION: us-east-2
+  AWS_BUCKET_NAME: "checkstyle-diff-reports"
+  USER_LOGIN: ${{ github.event.issue.user.login }}
+
+on:
+  issue_comment:
+    types: [ created]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+concurrency:
+  # run_id is guaranteed to be unique and present
+  group: ${{ github.run_id }}
+  cancel-in-progress: false
+
+jobs:
+  parse_comment:
+    if: |
+      (startsWith(github.event.comment.body, 'Github, generate report for ') ||
+       startsWith(github.event.comment.body, 'GitHub, generate report for ')) &&
+      github.event.issue.pull_request
```

Comment:
github.event.issue.pull_request
&& (startsWith(github.event.comment.body, 'Github, generate report for ')
    || startsWith(github.event.comment.body, 'GitHub, generate report for '))

Reply:
@relentless-pursuit , please place && to front, same for ||

Reply:
@relentless-pursuit , please put a bit more space before ||

To make it visually clear on priority of execution of such operations.

Reply:
done